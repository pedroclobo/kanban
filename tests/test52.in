a ANY+-*?="#$!_	()[]
a
q
