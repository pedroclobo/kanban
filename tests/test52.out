TO DO
IN PROGRESS
DONE
ANY+-*?="#$!_	()[]
